README.md

```diff
@@ -1,2 +1,5 @@
 # Juntos
 Juntos is a web application that lets you co watch streaming service such as youtube. More services to be added in future such as Netflix, Prime, HBO NOW ...etc.
+
+
+## <a href="prd/Juntos - PRD.pdf">PRD</a>
```

Comment:
Can we have a header named "Artefacts" or "Links". That way we can have a list of links under the same.

Reply:
Added header named "Artefacts"